docs/source/cpp/getting_started.rst

```diff
@@ -24,25 +24,27 @@ Getting Started
 The following articles demonstrate installation, use, and a basic understanding of Arrow.
 These articles will get you setup quickly using Arrow and give you a taste of what the
 library is capable of.
-Specifically, it contains: an installation and linking guide; documentation of conventions used 
-in the codebase and suggested for users; and tutorials, including: 
+Specifically, it contains: an installation and linking guide; documentation of conventions used
+in the codebase and suggested for users; and tutorials, including:
 
-* Building Arrow arrays and tabular structures 
+* Building Arrow arrays and tabular structures
 * Reading and writing Parquet, Arrow, and CSV files
 * Executing compute kernels on arrays
 * Reading and writing multi-file partitioned datasets
 
 Start here to gain a basic understanding of Arrow, and move on to the :doc:`/cpp/user_guide` to
-explore more specific topics and underlying concepts, or the :doc:`/cpp/api` to explore Arrow's 
-API. 
+explore more specific topics and underlying concepts, or the :doc:`/cpp/api` to explore Arrow's
+API.
 
 .. toctree::
 
+   overview
    build_system
    conventions
    tutorials/basic_arrow.rst
    tutorials/io_tutorial.rst
    tutorials/compute_tutorial.rst
    tutorials/datasets_tutorial.rst
-   
-   
+   acero_overview
```

Comment:
These should go in User's Guide -- it's all very technical, and not tutorial-shaped.